src/app/shared/services/base.service.ts

```diff
@@ -27,7 +27,6 @@ export abstract class BaseService {
     const apiUrl = this.apiUrlFor(endpoint);
 
     return this.http.post(apiUrl, body).pipe(
-      retry(1),
```

Comment:
@carryall Why we are removing it? may be it helps while app facing some net work issues as it will try to request again if it fails once? 🤔

Reply:
@junan but if there's an invalid request it's will request twice 🤔  I don't think in this case it's going to be useful and also it easier to test 😄